specs/patch/fixtures/fixtures-1.md

```diff
@@ -0,0 +1,283 @@
+Patch Fixtures 1
+================
+
+This file contains a bunch of fixtures for the Patch system.
+These are suitable both as mechanical test fixtures, and as examples for humans.
+
+### What's covered?
+
+All the fixtures in this file are operating on single blocks --
+the data can still be a tree (maps, lists, etc), but contains no links.
+(Other fixtures cover data that spans multiple blocks by use of links.)
+
+### What's the fixture format?
+
+This is the [testmark format](https://github.com/warpfork/go-testmark);
+it can be consumed programmatically.
+(Note that you probably _can't_ see these labels as this page is rendered on the website;
+you'll have to find the markdown [source](/specs/about/#source).)
+
+### This doesn't have to be JSON!
+
+We've used JSON here for the initial data, as well as the patch instructions and the results.
+However, this is a totally stylistic choice.  We're just using JSON here because it's convenient and it's human-readable.
+You can equally well express all of these objects in CBOR, or other [codecs](/docs/codecs/); it makes no difference to IPLD.
```

Comment:
nit: shouldn't you call these DAG-JSON and DAG-CBOR?

Reply:
I'm honestly not clear on if that's more helpful or less.

I do actually parse these with the json codec and not the dag-json codec in the first code I wrote using them, iirc.  But I suppose that's going to change the instant I add a fixture that crosses blocks!

Reply:
Well, to the rest of the world, JSON and CBOR are just the regular standards, without the detersministic encoding quailties that the DAG forms bring with them. Even if links are not involved, you probably do not mean just plain old JSON/CBOR.

Reply:
Added a mention that it's DAG-JSON and DAG-CBOR